pkg/api/common_test.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/grafana/grafana/pkg/services/searchusers/filters"
 	"github.com/grafana/grafana/pkg/services/sqlstore"
 	"github.com/grafana/grafana/pkg/services/sqlstore/mockstore"
+	startest "github.com/grafana/grafana/pkg/services/star/startest"
```

Comment:
I wonder how this became an import with a custom package name, which duplicates the original package name (`startest`)?